medicines/doc-index-updater/src/state_manager/redis.rs

```diff
@@ -30,7 +30,7 @@ impl From<RedisError> for MyRedisError {
 
 impl From<MyRedisError> for warp::Rejection {
     fn from(_e: MyRedisError) -> Self {
-        warp::reject::custom(MyRedisError::Other(anyhow!("Server Error")))
+        warp::reject::custom(MyRedisError::Other(anyhow!(_e)))
```

Comment:
maybe call `tracing::error` for that error here, but still return the Rejection as `Server Error` to avoid leaking error information to the end-user